powerauth-java-crypto/src/main/java/com/wultra/security/powerauth/crypto/lib/v4/dh/DhKem.java

```diff
@@ -116,6 +121,7 @@ public PublicKey convertBytesToPublicKey(byte[] publicKeyBytes) throws GenericCr
         try {
             return KEY_CONVERTOR.convertBytesToPublicKey(EcCurve.P384, publicKeyBytes);
         } catch (CryptoProviderException | InvalidKeySpecException e) {
+            logger.debug(e.getMessage(), e);
```

Comment:
I think that Coverity minds that the security exception `InvalidKeySpecException` may be swallowed. It is passed as a cause of GenericCryptoException, and static analysis may lose the context. Anyway, I am ok with this solution, since `GenericCryptoException` is a checked exception.